src/merkle/store.rs

```diff
@@ -110,20 +110,39 @@ impl MerkleStore {
 
     /// Returns the path for the node at `index` rooted on the tree `root`.
     ///
-    /// The path starts at the sibling of the target leaf.
-    ///
     /// # Errors
     ///
     /// This will return `NodeNotInStorage` if the element is not present in the store.
-    pub fn get_path(&self, root: Word, mut index: NodeIndex) -> Result<MerklePath, MerkleError> {
-        let mut path = Vec::with_capacity(index.depth().saturating_sub(1) as usize);
-        while index.depth() > 0 {
-            let sibling = index.sibling();
-            index.move_up();
-            let node = self.get_node(root, sibling)?;
-            path.push(node);
+    pub fn get_path(
```

Comment:
Would be better to add a documentation what is the first word of the tuple

Reply:
My idea was to add the return struct that we discussed on the original PR. But I can add docs too.